src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/Common.ps1

```diff
@@ -226,25 +108,27 @@ function Create-GalleryVM(
 	return $vm
 }
 
-function Create-SA(
-	[string] $resourceGroupName, 
-	[string] $location)
+function Delete-Vault($vault)
```

Comment:
Why is delete vault specific to VM tests? We can list all containers (irrespective of type) and do delete, right?